experiments/Atmos_SphereGCM/heldsuarez.jl

```diff
@@ -33,14 +33,16 @@ function init_heldsuarez!(state, aux, coords, t)
 end
 
 function config_heldsuarez(FT, N, resolution)
-    config = CLIMA.GCM_Configuration("HeldSuarez", N, resolution, domain_height, init_heldsuarez!,
-                                     ref_state=HydrostaticState(IsothermalProfile(T_initial),
+    config = CLIMA.Atmos_SphereGCM_Configuration("HeldSuarez", N,
```

Comment:
Maybe we can leave it at 
```suggestion
    config = CLIMA.Atmos_GCM_Configuration("HeldSuarez", N,
```
(don't think we would want a non-spherical GCM configuration anytime soon).

Reply:
Agreed.